identity-sdjwt/src/commonTest/kotlin/com/android/identity/sdjwt/SdJwtVcTest.kt

```diff
@@ -55,9 +51,7 @@ class SdJwtVcTest {
     private lateinit var issuerCert: X509Cert
     private lateinit var credential: KeyBoundSdJwtVcCredential
 
-    @Before
-    fun setup() {
-        Security.insertProviderAt(BouncyCastleProvider(), 1)
+    init {
```

Comment:
By moving this from a `@Before` function to init, we're losing the hermeticity we had before. Now, there's a single EphemeralStorage instance that's reused throughout all the tests, for instance. So if there are tables that might collide between tests, we could end up with flakey tests.

I'd there a reason we need a single instance, rather than a new one per test?

Reply:
OK, good point, moved this into the `provisionCredential()` call so it happens once per time the test is called.